tests/test_generator_tables.py

```diff
@@ -237,9 +311,9 @@ def test_mysql_column_types(generator: CodeGenerator) -> None:
         "simple_items",
         generator.metadata,
         Column("id", mysql.INTEGER),
-        Column("name", mysql.VARCHAR(255)),
-        Column("double", mysql.DOUBLE(1, 2)),
-        Column("set", mysql.SET("one", "two")),
+        Column("name", mysql.VARCHAR(255)),  # type: ignore[no-untyped-call]
```

Comment:
@agronholm are thse new ruff / pep8 standards we need to exclude?

Reply:
Why do we suddenly need these mypy exclusions?

Reply:
I thought you asked about Ruff / PEP 8 standards. But yeah, mypy keeps changing and sometimes it triggers errors it didn't before. And also this PR changes the minimum mypy version and removes the `mypy` extra, so maybe that's why?

Reply:
Ah, yes I see it here https://docs.sqlalchemy.org/en/20/orm/extensions/mypy.html

So, should we tell pre-commit to exclude this test for the time being?

Reply:
Why? I didn't think the impact was that bad – a few type ignore comments? We can also ignore this specific error category altogether if necessary.

Reply:
Yes, I prefer we do that. Otherwise, it will create inconsistencies in the test where some will have these annotations and others won't.